flink-runtime/src/test/java/org/apache/flink/runtime/deployment/ResultPartitionDeploymentDescriptorTest.java

```diff
@@ -109,7 +110,7 @@ public void testReleasedOnConsumptionFlag() {
 		for (ResultPartitionType partitionType : ResultPartitionType.values()) {
 			ResultPartitionDeploymentDescriptor partitionDescriptor = new ResultPartitionDeploymentDescriptor(
 				new PartitionDescriptor(resultId, partitionId, partitionType, numberOfSubpartitions, connectionIndex),
-				ResultPartitionID::new,
+				NettyShuffleDescriptorBuilder.newBuilder().buildLocal(),
```

Comment:
it might be a separate hotfix commit.

Reply:
The target change is not possible without this small change.

Reply:
yes, I misunderstood before.